lighthouse-plugin-ad-speed-insights/audits/ad-top-of-viewport.js

```diff
@@ -78,7 +79,7 @@ class AdTopOfViewport extends Audit {
       return auditNotApplicable(NOT_APPLICABLE.NO_ADS_VIEWPORT);
     }
 
-    const score = inViewport && topSlot.midpoint < 200 ? 0 : 1;
+    const score = inViewport && topSlot.midpoint < 100 ? 0 : 1;
 
     return {
       score,
```

Comment:
How about having a non binary score here? Something like:
```
const score = Math.min(topSlot.midpoint, 100) / 100
```

But maybe the score should be non-linear. Something like `Math.min((topSlot.midpoint / 100) ** 2, 1)`

Reply:
I like the idea but will defer to a later PR so I am able to thoroughly test against multiple pubs.